README.md

```diff
@@ -1 +1,10 @@
-# santasonic
\ No newline at end of file
+# santasonic
+
+TODOS:
+- wymyślić strukture repo. moja propozycja jest taka, żeby mieć folder projects,
+  w projects mieć już podfolderki, które najlepiej jakby były nazywane sprytnie. Np. prosty program, który blinkuje diodą żeby nazywał się np. simple_blink.
+  W środku każdego projekciku fajnie jakby znajdowały się dokładnie trzy rzeczy:
+  1. kod programu nazwany dowolnie np. simple_blink.py
+  2. folder z wymaganymi bibliotekami jeśli takie są
+  3. README.md z instrukcją podłączenia elementów jeśli to potrzebne
+- oprócz folderu projects fajnie by było mieć folder libraries, z dodatkowymi bibliotekami, również z bibliotekami napisanymi przez nas (fajnie jakby nasze były jakoś zaznaczone może).
```

Comment:
Generalnie są tutaj rzeczy, które pewnie ciężko zrobić - np wrzucić filmik, ale dużym plusem trzymania tego wszystkiego w repo, jest to, że to byłoby wszystko w repozytorium i nie byłoby jakiegoś pierdzielenia się

Reply:
Zastanawiam się jeszcze nad jedną rzeczą - ile my takich projekcików będziemy mieli? Max 5 pewnie, więc nie wiem czy to nie jest zbyt przekombinowana struktura.

Reply:
A no i w readme można na przykład dodać link do zdjęcia (które np. gdzieś będziemy w repo trzymali na twardo)